amps/ags/rm-automation/rm-automation-community-rest-api/src/main/java/org/alfresco/rest/rm/community/requests/gscore/api/HoldsAPI.java

```diff
@@ -287,4 +291,113 @@ public void deleteHoldChild(String holdId, String holdChildId)
     {
         deleteHoldChild(holdId, holdChildId, EMPTY);
     }
+
+    /**
+     * Starts a bulk process for a hold.
+     *
+     * @param holdBulkOperation The bulk operation details
+     * @param hold The identifier of a hold
+     * @param parameters The URL parameters to add
+     * @return The {@link HoldBulkOperationEntry} for the started bulk process
+     * @throws RuntimeException for the following cases:
```

Comment:
Generally we use (or extend) AlfrescoRuntimeException. I'm not sure whether that's what you've done or not here.